app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class RegistrationsController < Devise::RegistrationsController
+  layout 'internal'
+
+  protected
+
+  def after_update_path_for(_resource)
```

Comment:
@davithss your param name is `_resource` but you are using the `resource`. I think that you don't need this parameter here

Reply:
Thanks for advising me. It's must be `#after_update_path_for(resource) as devise docs says.` without `_`.